lib/tasks/certs.rake

```diff
@@ -3,6 +3,7 @@ require 'open3'
 namespace :certs do
   desc 'Remove invalid certs, set EXPIRING=true to also remove certs expiring within 30 days'
   task remove_invalid: :environment do
+    CertificateStore.instance.load_certs!(dir: Rails.root.join('config/certs'))
```

Comment:
Without this I get a bunch of errors running these rake tasks because the certificate store is empty. I copied this from a pattern I saw in other rake tasks in here

Reply:
also, since we have to do this for task, what if we used rake's task dependency and like defined a task for this?


```ruby
task load_certs: :environment do 
  CertificateStore.instance.load_certs!(dir: Rails.root.join('config/certs'))
end
# ...
task remove_invalid: :load_certs do
```

Reply:
Actually, I see the problem. I'll back these outs.

For reference, I was using `rails certs:remove_invalid`. That means this was not running:

```
# config/initializers/certificate_store.rb
unless File.basename($PROGRAM_NAME) == 'rake' && ARGV.any? { |arg| arg.start_with?('db:') }
  CertificateStore.instance.load_certs!
end
```